### PR TITLE
[Torch][PTQ] Mobilenet v2 save/load example 

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

Mobilenet v2 save/load example, possible to export to both onnx and openvino

### Reason for changes

Showcase for save/load model functionality

### Related tickets

129586

### Tests

test example is updated

Build test_examples/381/ finished successfully

### CODE REVIEW

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+fastdownload==0.0.7
+openvino-dev==2024.0
```
Comment (AlexanderDokuchaev) [inline examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt:2]:
```suggestion
openvino==2024.1.0
```

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+scikit-learn
```
Comment (AlexanderDokuchaev) [inline examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt:3]:
```suggestion
scikit-learn==1.4.2
```

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/quantize.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from typing import Tuple
+
+import torch
+
+import nncf
+from examples.post_training_quantization.torch.save_load_mobilenet_v2.common import QUANTIZED_CHECKPOINT_FILE_NAME
```
Comment (AlexanderDokuchaev) [inline examples/post_training_quantization/torch/save_load_mobilenet_v2/quantize.py:18]:
```suggestion
from common import QUANTIZED_CHECKPOINT_FILE_NAME
```
import from root of repository works only if set PYTHON_PATH

**Review by AlexanderDokuchaev — CHANGES_REQUESTED** [review]

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/quantize.py (hunk 3 above)
Comment (daniil-lyakhov) [reply examples/post_training_quantization/torch/save_load_mobilenet_v2/quantize.py]: Fixed

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt (hunk 2 above)
Comment (daniil-lyakhov) [reply examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt]: 1.4.2 version starts from python3.9, I used the commonly used 1.2.2 version

File: examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt (hunk 1 above)
Comment (daniil-lyakhov) [reply examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt]: Done

**Pull request closed without merging**